### DISCUSSION TITLE
Legend for radius

### DISCUSSION THREAD

**Discussion by nobarany** [observablehq/plot#1535] [Q&A] [answered]
Is there a way to produce a legend for radius? Setting the legend=true like the following doesn't seem to do anything.
```
    r: {
      legend: true,
      label: 'label',
      range: [3, 6],
      domain: [1, 1000],
      type: 'linear',
    },
```

**Comment by mbostock** [comment, marked as answer]
Please upvote this feature request: https://github.com/observablehq/plot/issues/236 Thanks!

**Reply by nobarany** [reply]
thank you!
